kickstart/PXELinux.erb

```diff
@@ -24,6 +24,8 @@ LABEL linux
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
     <% elsif @host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7 -%>
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> network ks.sendmac
+    <% elsif @host.operatingsystem.name =~ /.*Atomic.*/ -%>
+    APPEND initrd=<%= @host.medium.path %>isolinux/initrd.img repo=<%= @host.medium.path %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
```

Comment:
This will also fail when safemode is on: `Host#medium` isn't accessible from the jail, and neither is `Medium#path`